packages/vscode-lit-plugin/src/test/fixtures/completions.ts

```diff
@@ -0,0 +1,28 @@
+// Pretending this is the Lit html function
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare const html: any;
+
+/** An element to test autocomplete with. */
+class CompleteMe extends HTMLElement {
+	/** Docs for prop 1. */
+	prop1 = "";
+	/** Docs for prop 2. */
+	prop2 = "";
+	/** Docs for prop 3. */
+	prop3 = "";
+}
+customElements.define("complete-me", CompleteMe);
+declare global {
+	interface HTMLElementTagNameMap {
+		"complete-me": CompleteMe;
+	}
+}
+
+html`
+	<complete-me
```

Comment:
Why's this formatted this way? Is the completion done at the end of the first line? Maybe add a comment to day that.

Reply:
Added comment